2020/latex/ISMIR2020template.tex

```diff
@@ -87,7 +88,11 @@ \section{Introduction}\label{sec:introduction}
 This template can be downloaded from the ISMIR \conferenceyear\ web site (\texttt{http://ismir\conferenceyear.ismir.net}).
 %
 \section{Paper Length \& File Size}
+\ifdefined\lbd
+Late-Breaking Demo papers may have a maximum of 2 pages. Please note that extended abstract is not a conference paper. Therefore, it will not be part of the official ISMIR 2020 proceedings, and should not be cited as such.

```

Comment:
Should be \conferenceyear instead of 2020!

Reply:
Fixed with ff93c4232c35b9bd88921206c34de599b6a7084b